payjoin/src/send/error.rs

```diff
@@ -343,7 +358,7 @@ impl fmt::Debug for ResponseError {
 pub enum WellKnownError {
     Unavailable(String),
     NotEnoughMoney(String),
-    VersionUnsupported(String),
+    VersionUnsupported(String, Vec<u64>),
```

Comment:
we can use `u32` or something even smaller here to save some bytes

Reply:
I agree that even `u8` would be fine for this enum, but `Value` only implements [`as_u64`](https://docs.rs/serde_json/latest/serde_json/enum.Value.html#method.as_u64), so we're going to be using that memory no matter what and mapping u64 to u8 requires more code to maintain https://chat.openai.com/c/8b7f115c-390c-4185-8582-9a94e41fe967. For those reasons I think this data structure is optimal for our purposes